runtime/vm/ContinuationHelpers.cpp

```diff
@@ -91,43 +91,42 @@ createContinuation(J9VMThread *currentThread, j9object_t continuationObject)
 }
 
 j9object_t
-synchronizeWithConcurrentGCScan(J9VMThread *currentThread, j9object_t continuationObject, J9VMContinuation *continuation)
+synchronizeWithConcurrentGCScan(J9VMThread *currentThread, j9object_t continuationObject, ContinuationState volatile *continuationStatePtr)
```

Comment:
put 3 short comments ahead of each of 3 high level parts of this method (or one comment ahead of the method?):
- Set pending state and carrier id
- Wait till potentially concurrent GC scans are complete
- Remove pending state